src/components/SocialIcons.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div id="social">
+    <a
+      href="https://github.com/aecworks"
+      :class="gIconExpanded ? 'social-icon--container-active' : 'social-icon--container'"
+      class="transition"
+      @mouseover="gHandleExpand"
+      @mouseout="gHandleCollapse"
+    >
+      <img
+        alt="Github"
+        :class="gIconExpanded ? 'social-icon-active' : 'social-icon'"
+        src="@/assets/images/social-icons/github.svg"
+      />
+      <div :class="gIconExpanded ? 'social-icon--text-active' : 'social-icon--text'">Github</div>
+    </a>
+    <a
+      href="https://twitter.com/gtalarico"
+      :class="tIconExpanded ? 'social-icon--container-active' : 'social-icon--container'"
+      class="transition"
+      @mouseover="tHandleExpand"
+      @mouseout="tHandleCollapse"
+    >
+      <img
+        alt="Twitter"
+        :class="tIconExpanded ? 'social-icon-active' : 'social-icon'"
+        src="@/assets/images/social-icons/twitter.svg"
+      />
+      <div :class="tIconExpanded ? 'social-icon--text-active' : 'social-icon--text'">Twitter</div>
+    </a>
+    <a
+      href="https://twitter.com/gtalarico"
+      :class="aIconExpanded ? 'social-icon--container-active' : 'social-icon--container'"
+      class="transition"
+      @mouseover="aHandleExpand"
+      @mouseout="aHandleCollapse"
+    >
+      <img
+        alt="About"
+        :class="aIconExpanded ? 'social-icon-active' : 'social-icon'"
+        src="@/assets/images/social-icons/about.svg"
+      />
+      <div :class="aIconExpanded ? 'social-icon--text-active' : 'social-icon--text'">About</div>
+    </a>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'SocialIcons',
+  data() {
+    return {
+      gIconExpanded: false,
+      tIconExpanded: false,
+      aIconExpanded: false,
+    }
+  },
+  methods: {
+    gHandleExpand: function () {
```

Comment:
I would suggest removing all JS related to hover. Why not do it with just CSS?

See how hover for buttons are currently setup. This would remove all mouseover, mouse out, and methods, these state vars.

https://github.com/aecworks/aec.works-web/blob/9e4046cc62b8185d118da1ba430f718ceb2a34d6/src/assets/scss/components/buttons.scss#L22

Eg.
```
.social-icon {
     ...
    @extend .border-thin;
    transition: box-shadow 200ms;
    &:hover{
        @include shadow-color($dark);
    }
    &:active {
        box-shadow: 0 0 $dark;
    }
```